### PR TITLE
Added ability to retrieve bit depth from media streams for lossless encodings

### PR DESCRIPTION
**Pull request by tombogle**
Addresses https://github.com/rosenbjerg/FFMpegCore/issues/358

### CODE REVIEW

File: FFMpegCore.Test/Resources/TestResources.cs [hunk 1]
```diff
@@ -21,5 +21,9 @@ public static class TestResources
         public static readonly string PngImage = "./Resources/cover.png";
         public static readonly string ImageCollection = "./Resources/images";
         public static readonly string SrtSubtitle = "./Resources/sample.srt";
+        public static readonly string AiffAudio = "./Resources/sample3aiff.aiff";
```
Comment (rosenbjerg) [inline FFMpegCore.Test/Resources/TestResources.cs:24]: Could you resize or shorten the `sample3aiff.aiff` test file?

Comment (rosenbjerg) [reply FFMpegCore.Test/Resources/TestResources.cs:24]: To maybe around 2-3mb or less

Comment (tombogle) [reply FFMpegCore.Test/Resources/TestResources.cs:24]: Done

**Comment by rosenbjerg** [comment]
Awesome, thank you

**Comment by tombogle** [comment]
Thank you! Do you have an idea how soon this might be released as an updated NuGet package?

**Comment by rosenbjerg** [comment]
@tombogle version 5.0.0 has been released :)

**Comment by tombogle** [comment]
> @tombogle version 5.0.0 has been released :)

Thank you!!!

**Pull request merged**